jest.config.js

```diff
@@ -7,7 +7,7 @@ const config = require('./.config/jest.config');
 module.exports = {
   // Jest configuration provided by Grafana scaffolding
   ...config,
-  setupFilesAfterEnv: ['<rootDir>/src/test/jest-setup.tsx'],
+  setupFilesAfterEnv: ['<rootDir>/src/test/encoder-workaround.tsx', '<rootDir>/src/test/jest-setup.tsx'],
```

Comment:
I'd just include the encoder-workaround in `jest-setup.tsx`.

Because there are so _many_ different ways to set-up a jest environment I've been working towards condensing as much as we can so the `src/test/jest-setup.tsx` file is the singular point of reference so it's easier to reason with rather than going:

1. Let me check what's in package.json
2. Let me check what's in `jest.config.js`
3. Let me check what folders exist named `__mocks__` and what's in them
4. Let me check what environment variables are
5. Let me check what's imported / mocked in individual tests

etc. 😄

Reply:
I moved it there, but if I do that it will fail if it's not present in the `grafana/ui` mock so I had to add it there as well: https://github.com/grafana/synthetic-monitoring-app/pull/815#discussion_r1635279546